modules/test-structure/test_structure.go

```diff
@@ -72,6 +72,13 @@ func CopyTerraformFolderToTemp(t testing.TestingT, rootFolder string, terraformM
 		return filepath.Join(rootFolder, terraformModuleFolder)
 	}
 
+	fullTerraformModuleFolder := filepath.Join(rootFolder, terraformModuleFolder)
+
+	_, err := files.DirExistsE(fullTerraformModuleFolder)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Same issue here... Shouldn't you be using the bool returned?

Reply:
fixed as well